bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/ItemUtil.java

```diff
@@ -75,14 +75,18 @@ public static void assertValidItemName(String itemName) throws IllegalArgumentEx
         }
     }
 
-    public static State convertToAcceptedState(final State state, final Item item) {
+    public static State convertToAcceptedState(State state, Item item) {
+        if (state == null) {
```

Comment:
@martinvw I think we should not silently swallow this situation, but log an error (incl. stacktrace) nonetheless to highlight that there is some bug somewhere in the code as we should never get here.